src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+body,
+.search-text {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  color: #3d4e61;
+}
+
+.search-form {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search-text {
+  width: 100%;
+  outline: none;
+  font-weight: bold;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-text::placeholder {
+  font-weight: normal;
+}
+
+.large {
+  font-size: 16px;
+  padding: 23px 23px 23px 62px;
+}
+
+.little {
+  font-size: 14px;
+  padding: 9px 8px 11px 33px;
+}
+
+.search-text:hover {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-text:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
+
+.search-icon-big {
+  position: absolute;
+  top: 26px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  z-index: 2;
+}
```

Comment:
You have some common styles for `search-icon-big` and `search-icon-small`
Move them to common class, same as you did with forms and inputs